Jenkinsfile_CNP

```diff
@@ -48,9 +48,9 @@ withPipeline(type, product, component) {
                 Using PREVIEW_ENVIRONMENT_NAME: ${env.PREVIEW_ENVIRONMENT_NAME}
                 Using NONPROD_ENVIRONMENT_NAME: ${env.NONPROD_ENVIRONMENT_NAME}""".stripIndent()
     }
-
+    
```

Comment:
This change is littered with 👾

Reply:
There's no diff really. The merge is just adding whitespace aka space invaders

Reply:
🤦 I didn't get the reference :)